dnf/cli/commands/repolist.py

```diff
@@ -113,6 +113,8 @@ def run(self):
         on_hiend = term.MODE['normal']
         tot_num = 0
         cols = []
+        if not repos:
+            print(_("No repos available"))
```

Comment:
A couple of nitpicks: 

We prefer single quotes over double quotes. 
I think 'No repositories available.' will be better.
And this should go to std.err. Therefore use `logger.warn(...)`.